dockers/docker-database/supervisord.conf

```diff
@@ -12,18 +12,10 @@ stdout_logfile=syslog
 stderr_logfile=syslog
 
 [program:redis-server]
-command=/usr/bin/redis-server /etc/redis/redis.conf
+command=/bin/bash -c "{ [[ -s /var/lib/redis/dump.rdb ]] || rm -f /var/lib/redis/dump.rdb; } && exec /usr/bin/redis-server /etc/redis/redis.conf"
 priority=2
 autostart=true
 autorestart=false
 stdout_logfile=syslog
 stderr_logfile=syslog
 
-[program:configdb-load.sh]
```

Comment:
configdb-load is also used in virtual switch, can you change that too?

Reply:
I would like keep behavior unchange on vs side, and visit the warm-reboot feature later.
Also applied to P4 side.

---
In reply to: [235583086](https://github.com/Azure/sonic-buildimage/pull/2287#discussion_r235583086) [](ancestors = 235583086)